packages/tenant-process/test/db.test.ts

```diff
@@ -78,6 +103,784 @@ describe("database test", () => {
+          offset: 0,
+          limit: 2,
+        });
+        expect(tenantsByName.results.length).toBe(2);
+      });
+      it("Should get consumers (pagination: offset, limit)", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+
+        const descriptor1: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService1: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor1],
+          producerId: organizationProducerId,
+        };
+        await addOneEService(eService1, eservices);
+
+        const agreementEservice1 = getMockAgreement({
+          eServiceId: eService1.id,
+          descriptorId: descriptor1.id,
+          producerId: organizationProducerId,
+          consumerId: consumerId1,
+        });
+        await addOneAgreement(agreementEservice1, agreements);
+
+        await addOneTenant(tenant2, postgresDB, tenants);
+
+        const descriptor2: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService2: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "B",
+          descriptors: [descriptor2],
+          producerId: organizationProducerId,
+        };
+        await addOneEService(eService2, eservices);
+
+        const agreementEservice2 = getMockAgreement({
+          eServiceId: eService2.id,
+          descriptorId: descriptor2.id,
+          producerId: organizationProducerId,
+          consumerId: consumerId2,
+        });
+        await addOneAgreement(agreementEservice2, agreements);
+
+        await addOneTenant(tenant3, postgresDB, tenants);
+
+        const descriptor3: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService3: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "C",
+          descriptors: [descriptor3],
+          producerId: organizationProducerId,
+        };
+        await addOneEService(eService3, eservices);
+
+        const agreementEservice3 = getMockAgreement({
+          eServiceId: eService3.id,
+          descriptorId: descriptor3.id,
+          producerId: organizationProducerId,
+          consumerId: consumerId3,
+        });
+        await addOneAgreement(agreementEservice3, agreements);
+
+        const tenantsByName = await readModelService.getConsumers({
+          consumerName: undefined,
+          producerId: organizationProducerId,
+          offset: 2,
+          limit: 3,
+        });
+        expect(tenantsByName.results.length).toBe(1);
+      });
+    });
+    describe("getProducers", () => {
+      const organizationProducerId1 = uuidv4();
+      const organizationProducerId2 = uuidv4();
+      const organizationProducerId3 = uuidv4();
+      const mockTenant = getMockTenant();
```

Comment:
```suggestion
```
Same as before, I don't think these are needed, we can replace these with `uuidv4()`